EnglishFilter/sections/specific.txt

```diff
@@ -284,6 +284,7 @@ outlookindia.com,cnbctv18.com,internationalnewsandviews.com,additudemag.com,moto
 ! 24.08.21 changed the rule after two weeks
 techpowerup.com##a[href$=".xlv"] > img
 !
+softvela.us##.theiaStickySidebar > #custom_html-3
```

Comment:
Hello @RadicalNoise
`#custom_html-3` is always being used in sidebar, please just add the domain to this existing rule:
`hwnews.in,collegelearners.com,watchmalcolminthemiddle.com,romadd.com###custom_html-3`